src/java.base/share/classes/sun/launcher/LauncherHelper.java

```diff
@@ -335,6 +317,11 @@ private static void printLocales() {
     }
 
     private static void printSystemMetrics() {
+        // only Linux supported
+        if (!OperatingSystem.isLinux()) {
+            abort(null, "java.launcher.bad.option");
```

Comment:
```suggestion
            abort(null, "java.launcher.bad.option", "system");
```

maybe the `isLinux` check can be moved to `showSettings` and fall through to abort with the unmatched case.